spec/factories/allocations.rb

```diff
@@ -7,7 +7,7 @@
     end
 
     created_by_name do
-      "#{Faker::Name.first_name} #{Faker::Name.last_name}"
+      "#{Faker::Name.first_name} #{Faker::Name.last_name.titleize}"
```

Comment:
It would be good to put a comment somewhere to explain all these - otherwise looks good